packages/utils/src/libp2p/index.ts

```diff
@@ -16,6 +19,30 @@ export function selectRandomPeer(peers: Peer[]): Peer | undefined {
   return peers[index];
 }
 
+/**
+ *
+ * @param ping The libp2p ping service's ping function
```

Comment:
As discussed, I'd just re-use the keep alive functionality at this point in time.